uwp missing

```
    (CoreCompile target) -> 
         D:\a\_work\1\s\src\Essentials\src\Connectivity\Connectivity.uwp.cs(8,52): error CS0535: 'ConnectivityImplementation' does not implement interface member 'IConnectivity.ConnectionProfiles' [D:\a\_work\1\s\src\Essentials\src\Essentials.csproj]
         D:\a\_work\1\s\src\Essentials\src\Connectivity\Connectivity.uwp.cs(8,52): error CS0535: 'ConnectivityImplementation' does not implement interface member 'IConnectivity.NetworkAccess' [D:\a\_work\1\s\src\Essentials\src\Essentials.csproj]
         D:\a\_work\1\s\src\Essentials\src\Connectivity\Connectivity.uwp.cs(8,52): error CS0736: 'ConnectivityImplementation' does not implement instance interface member 'IConnectivity.StartListeners()'. 'ConnectivityImplementation.StartListeners()' cannot implement the interface member because it is static. [D:\a\_work\1\s\src\Essentials\src\Essentials.csproj]
         D:\a\_work\1\s\src\Essentials\src\Connectivity\Connectivity.uwp.cs(8,52): error CS0736: 'ConnectivityImplementation' does not implement instance interface member 'IConnectivity.StopListeners()'. 'ConnectivityImplementation.StopListeners()' cannot implement the interface member because it is static. [D:\a\_work\1\s\src\Essentials\src\Essentials.csproj]
```

--- a/src/Essentials/src/Connectivity/Connectivity.tizen.cs
+++ b/src/Essentials/src/Connectivity/Connectivity.tizen.cs
@@ -40,7 +40,7 @@ namespace Microsoft.Maui.Essentials.Implementations
 			OnConnectivityChanged();
 		}
 
-		static NetworkAccess PlatformNetworkAccess
+		public NetworkAccess PlatformNetworkAccess
 		{
 			get
 			{
@@ -58,7 +58,7 @@ namespace Microsoft.Maui.Essentials.Implementations
 			}
 		}
 
-		static IEnumerable<ConnectionProfile> PlatformConnectionProfiles
+		public IEnumerable<ConnectionProfile> PlatformConnectionProfiles
 		{
 			get
 			{
@@ -66,14 +66,14 @@ namespace Microsoft.Maui.Essentials.Implementations
 			}
 		}
 
-		static void StartListeners()
+		public void StartListeners()
 		{
 			Permissions.EnsureDeclared<Permissions.NetworkState>();
 			ConnectionManager.ConnectionTypeChanged += OnChanged;
 			GetProfileListAsync();
 		}
 
-		static void StopListeners()
+		public void StopListeners()
 		{
 			ConnectionManager.ConnectionTypeChanged -= OnChanged;
 		}

--- a/src/Essentials/src/Connectivity/Connectivity.tizen.cs
+++ b/src/Essentials/src/Connectivity/Connectivity.tizen.cs
@@ -40,7 +40,7 @@ namespace Microsoft.Maui.Essentials.Implementations
 			OnConnectivityChanged();
 		}
 
-		public NetworkAccess PlatformNetworkAccess
+		public NetworkAccess NetworkAccess
 		{
 			get
 			{
@@ -58,7 +58,7 @@ namespace Microsoft.Maui.Essentials.Implementations
 			}
 		}
 
-		public IEnumerable<ConnectionProfile> PlatformConnectionProfiles
+		public IEnumerable<ConnectionProfile> ConnectionProfiles
 		{
 			get
 			{

--- a/src/Essentials/src/Connectivity/Connectivity.uwp.cs
+++ b/src/Essentials/src/Connectivity/Connectivity.uwp.cs
@@ -7,16 +7,16 @@ namespace Microsoft.Maui.Essentials.Implementations
 {
 	public partial class ConnectivityImplementation : IConnectivity
 	{
-		static void StartListeners() =>
+		public void StartListeners() =>
 			 NetworkInformation.NetworkStatusChanged += NetworkStatusChanged;
 
 		static void NetworkStatusChanged(object sender) =>
 			OnConnectivityChanged();
 
-		static void StopListeners() =>
+		public void StopListeners() =>
 			 NetworkInformation.NetworkStatusChanged -= NetworkStatusChanged;
 
-		static NetworkAccess PlatformNetworkAccess
+		public NetworkAccess NetworkAccess
 		{
 			get
 			{
@@ -39,7 +39,7 @@ namespace Microsoft.Maui.Essentials.Implementations
 			}
 		}
 
-		static IEnumerable<ConnectionProfile> PlatformConnectionProfiles
+		public IEnumerable<ConnectionProfile> ConnectionProfiles
 		{
 			get
 			{

